datasets/br/bcb_disqualified_persons/crawler.py

```diff
@@ -0,0 +1,90 @@
+from typing import List
+from zavod import Context, helpers as h
+
+
+def fetch_data(context: Context) -> List[dict]:
+    """
+    Fetches data from the website, or raises an exception on failure.
+
+    :param context: The context object.
+    :return: List of dicts containing data.
+    """
+    response = context.fetch_json(context.data_url)
+    if "value" not in response:
+        context.log.error("Value not found in JSON")
+        return []
+    return response["value"]
+
+
+def create_entity(input_dict: dict, context: Context):
+    """
+    Creates an entity from the raw data.
+
+    :param input_dict: Data dict extracted from source.
+                       Should have at least following keys:
+                        - CPF
+                        - Nome
+    :param context: The context object.
+    """
+    entity = context.make("Person")
+    tax_number = input_dict["CPF"]
+    entity.id = context.make_id(tax_number)
+    entity.add("name", input_dict["Nome"])
+    entity.add("taxNumber", tax_number)
+    entity.add("country", "br")
+    entity.add('topics', 'corp.disqual')
+
+    return entity
+
+
+def create_sanction(input_dict: dict, entity, context: Context):
+    """
+    Creates the sanction for a given entity and the raw data.
+
+    :param input_dict: Data dict extracted from source.
+                       Should have at least following keys:
+                        - PAS
+                        - Prazo_em_ano
+                        - Inicio_do_cumprimento
+                        - Prazo_final_penalidade
+    :param entity: Entity the sanction refers to.
+    :param context: The context object.
+    """
+
+    sanction = h.make_sanction(context, entity)
+    sanction.add(
+        "program", "Brazil's Central Bank General Register of Disqualified Persons"
+    )
+    sanction.add("authority", "Brazil's Central Bank")
+
+    # The ID of the process
+    sanction.add(
+        "description",
+        "Administrative Sanctioning Process Number: {}".format(input_dict.pop("PAS")),
+    )
```

Comment:
Would it make sense to add the PAS id as the key argument to the `make_sanction`? That way each individual process would have its own sanction entity and the dates wouldn't get bagged together.

I noticed this by checking the number of rows in the table and trying to understand why the number of persons doesn't match the number of rows - it's because one person can be the subject of multiple investigations.

Reply:
I didn't find the key parameter in the Sanction schema https://followthemoney.tech/explorer/schemata/Sanction/

Reply:
I meant the key argument shown here https://zavod.opensanctions.org/helpers/#zavod.helpers.make_sanction
like `h.make_sanction(context, entity, key=pas)`

Reply:
I'll add that, should I keep the `description`? I added it because I couldn't find where to put the `PAS`, but now it might be irrelevant